.changeset/strange-items-clap.md

```diff
@@ -0,0 +1,14 @@
+---
+'@solid-design-system/components': major
+---
+
+!Breaking Change
+
+implement new sd-flipcard:
+
+- remove 'activation' feature.
+- Include a dedicated button to trigger the flip (clicking the card and hovering will no longer trigger a flip).
+- remove 'empty' variant and set 'primary' as the default variant.
+- add 'placement' feature to set the position of the default slots for both the front and back cards.
+- simplify variants to remove any positioning logic (eg: gradient-light-top -> gradient-light)
+- update tests and templates.
```

Comment:
```suggestion
!Breaking Change

Implement new sd-flipcard concept:

- Remove 'activation' feature.
- Include a dedicated button to trigger the flip (clicking the card and hovering will no longer trigger a flip).
- Remove 'empty' variant and set 'primary' as the default variant.
- Add 'placement' feature to set the position of the default slots for both the front and back cards.
- Simplify variants to remove any positioning logic (eg: gradient-light-top -> gradient-light)
- Update tests and templates.
```

Reply:
@Vahid1919 I've approved the PR but please do not merge yet as per [Mario's request](https://github.com/solid-design-system/solid/pull/1703#discussion_r1869887585).